templates/customers.json

```diff
@@ -110,6 +110,31 @@
               ]
             ]
           ],
+          "conditionally-change-invoice-method": [
```

Comment:
Could you add a comment here describing why we have to add this "business logic"?